utils/gdb_qemu/gdb_qemu/src/parent.rs

```diff
@@ -7,7 +7,7 @@ use std::{
     time::{Duration, SystemTime},
 };
 
-use anyhow::{anyhow, Result};
+use anyhow::{Result, anyhow};
```

Comment:
Are these not included in the format check yet? Or is this fmt newer than what we have currently?

Reply:
Seems they were skipped. I've just removed the exemption. https://github.com/AFLplusplus/LibAFL/blob/23185b642b43e47156081beb50ae184e83d7a694/utils/libafl_repo_tools/src/main.rs#L289